lector/resources/raw/io.github.BasioMeusPuga.Lector.metainfo.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 BasioMeusPuga <https://github.com/BasioMeusPuga> -->
```

Comment:
They recommend an email address here instead of the URL for your GitHub profile, but I'm not sure how much that matters.